src/dtos/git-file-object-content.ts

```diff
@@ -3,8 +3,8 @@ import { ApiModelProperty } from "@nestjs/swagger";
 import { GitObjectContent } from "./git-object-content";
 
 export class GitFileObjectContent extends GitObjectContent {
-  @ApiModelProperty({ type: String })
-  public content: string;
+  @ApiModelProperty({ type: String, required: false })
```

Comment:
I think here its better to do have 2 class one with content one without. that way you don't have it potentially undefined even if you know it isn't

Reply:
Yeah good point

Reply:
I did that for `GitCommit` and `GitCommitRef` for refernece